docs/xml-sitemaps.md

```diff
@@ -0,0 +1,54 @@
+# XML Sitemaps
+
+XML Sitemaps are a means of providing key content and additional meta data around the frequency of updates on your website to search engines.
+
+The functionality for this component is provided by the [MSM Sitemap](https://github.com/humanmade/msm-sitemap) plugin for WordPress.
+
+Performance problems are solved by generating the XML ahead of time and storing it in the database.
+
+## Indexing content by year
+
+In cases where you have a lot of content you can split your sitemap up into multiple files by year using the following code:
+
+```php
+add_filter( 'msm_sitemap_index_by_year', '__return_true' );
+```
+
+## Modifying a sitemap entry
+
+In order to modify or add extra data to a URL entry in the sitemap XML you can use the `msm_sitemap_entry` filter. Functions attached to this filter can access the current post using `$GLOBALS['post']` or template tags such as `get_the_permalink()`.
```

Comment:
I'd say `get_post()` rather than `$GLOBALS['post']`, pretty sure `get_post()` returns the global one right?

Reply:
It does yeah. Just wanted to make it clear `$post` was in the global scope but `get_post()` does look neater